js/DisableUiComponent.js

```diff
@@ -37,10 +37,11 @@ DisableUiComponent.prototype.disable = function (target) {
         // Disable links (uses .js-disable as any existing disabled)
         $this.find(LINK_ELEMENTS)
             .on('click', preventDefaultAndStopPropagation)
-            .addClass('js-disabled u-cursor-not-allowed');
+            .addClass('js-disabled u-cursor-not-allowed')
+            .prop('aria-disabled', 'true');
```

Comment:
Minor point, we use a mix of `attr` and `prop` here. I'd stick to `attr` unless we are specifically changing a `prop`